controllers/endpoint.controller.js

```diff
@@ -32,5 +33,19 @@ exports.getArticlesByID = (req, res, next) => {
     });
 };
 
+exports.getArticles = (req, res, next) => {
+    return selectArticles()
+    .then((articles) => {
+        if(articles.length === 0) {
+            return Promise.reject( { status: 404, msg: "No articles found"})
```

Comment:
We don't need to actually throw an error here. If the articles array that comes back is empty, it's not an error, it would just indicate that the table hadn't been seeded correctly.